.gitignore

```diff
@@ -55,3 +55,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Sqlite databases
+*.db
```

Comment:
In fact these are not for tests, which are cleaning properly. This is for the examples. The `app.py` example has its instance directory inside the `examples` directory for two reasons. The first one is that the directory was not created if it did not exist. The second reason is that every module's example is named `app,py` and thus has the same instance director. I had to clean it manually each time in order to be sure that the database was in a clean state.
Thus I create an instance directory for each example in the examples directory.